dev/tuple_helper/tuple_iteration.h

```diff
@@ -35,7 +35,7 @@ namespace sqlite_orm {
 
 #if defined(SQLITE_ORM_FOLD_EXPRESSIONS_SUPPORTED) && defined(SQLITE_ORM_IF_CONSTEXPR_SUPPORTED)
         template<bool reversed = false, class Tpl, size_t... Idx, class L>
-        void iterate_tuple(const Tpl& tpl, std::index_sequence<Idx...>, L&& lambda) {
+        void iterate_tuple(Tpl& tpl, std::index_sequence<Idx...>, L&& lambda) {
```

Comment:
why did you remove `const`? if we don't modify the tuple then we don't ned it as mutable

Reply:
`compound_operator`'s contructor [sets](https://github.com/fnc12/sqlite_orm/blob/9813ec16425e3e4f070a7c22ddafb5cfa16d792f/dev/select_constraints.h#L121) the `highest_level` flag of the single select-statements.